libbeat/outputs/elasticsearch/output.go

```diff
@@ -123,17 +121,14 @@ func (out *elasticsearchOutput) init(
 
 // loadTemplate checks if the index mapping template should be loaded
 // In case template loading is enabled, template is written to index
-func loadTemplate(config Template, clients []mode.ProtocolClient) {
+func loadTemplate(config Template, client *Client) {
```

Comment:
with multiple elasticsearch hosts being configured, old implementation used to use first configured host only (drawback: what if first configured host is unresponsive). This change ensures 'loadTemplate' is executed per configured host.

There is a big chances of clients racing to configure the template. Is this potentially a problem? If so, consider using a mutex to linearize template loading. Still, every client must have the chance to load(check) the template, no matter the result for other clients to load the template.